docker/staging/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: '3.3'
+services:
+  coordinator:
+    build:
+      context: "../../"
+      dockerfile: "docker/Dockerfile"
+    networks:
+      - labgrid_net
+    tty: true
+    entrypoint: "crossbar start"
+  client:
+    build:
+      context: "../../"
+      dockerfile: "docker/Dockerfile"
+    networks:
+      - labgrid_net
+    tty: true
+    stdin_open: true
+    entrypoint: "/bin/bash"
+  exporter:
+    build:
+      context: "../../"
+      dockerfile: "docker/Dockerfile"
+    volumes:
+      - "./exporter-conf:/opt/exporter-conf"
+    devices:
+      - "/dev/ttyUSB0:/dev/ttyUSB0"
+    networks:
+      - labgrid_net
+    entrypoint: /opt/exporter-conf/entrypoint.sh
+    depends_on:
+      - coordinator
+    tty: true
+    stdin_open: true
+    privileged: true
+
+networks:
+  labgrid_net:
+    driver: bridge
+    driver_opts:
+      com.docker.network.bridge.name: labgrid_bridge
+    ipam:
+      driver: default
+      config:
+        - subnet:  172.10.238.0/24
```

Comment:
Ouch, would it be possible to avoid hardcoding IP addresses?  It can conflict with existing use of private IP addresses (the company I work for uses this range internally...).